guides/hiring/remote_test/csharp_marking_guide.md

```diff
@@ -0,0 +1,28 @@
+# Marking guide
+
+60 marks in total. There are no half marks. 80% (48 marks) required to pass.
```

Comment:
What marks did we remove as they were too specific? Perhaps there are some language agnostic alternatives?